src/java/us/kbase/mobu/tester/ConfigLoader.java

```diff
@@ -38,6 +38,11 @@ public ConfigLoader(Properties props, boolean testMode,
         if (authUrl == null)
             throw new IllegalStateException("Error: 'auth_service_url' parameter is not set in " +
                     configPathInfo);
+        endPoint = props.getProperty("kbase_endpoint");
```

Comment:
moved this check up so that kb-sdk will fail earlier if the `kbase_endpoint` property is not set